gcp-deployer/README.md

```diff
@@ -104,3 +104,24 @@ The yaml is unmarshalled into a [machinesetup.configList](../cloud/google/machin
 Right now, the startup scripts must be bash scripts. 
 There is a set of environment variables that is concatenated to the beginning of the startup script.
 You can find them [here](../cloud/google/metadata.go) and use them in your startup script as needed.
+
+## Contributing
+
+### Building
+To build a binary, run the following command from this folder.
+
+```
+go build .
+```
+
+### Testing
+To run the unit tests, run the following command from the parent of this folder.
+
+```
+go test ./...
+```
+
+To run the integration tests, run the following command from this folder.
+```
+go test -tags=integration -v
```

Comment:
Yea I think a minimal mention of the steps is fine, because it is well documented already. They have probably done some manual testing already so it's possible that they already know that is necessary. The contributing guide I linked to is also in the gcp-deployer folder and has it's own testing section with manual tests steps for the gcp-deployer.